asm/f64/asm_test.go

```diff
@@ -65,3 +68,48 @@ func nonStridedWrite(x []float64, inc int) bool {
 	}
 	return false
 }
+
+type incSet struct {
+	x, y int
+}
+
+func genInc(inc []int) []incSet {
+	n := len(inc)
+	is := make([]incSet, n*n)
+	for x := range inc {
+		for y := range inc {
+			is[x*n+y] = incSet{inc[x], inc[y]}
+		}
+	}
+	return is
+}
+
+type incToSet struct {
+	dst, x, y int
+}
+
+func genIncTo(inc []int) []incToSet {
+	n := len(inc)
+	is := make([]incToSet, n*n)
+	for dst := range inc {
+		for x := range inc {
+			for y := range inc {
+				is[x*n+y] = incToSet{inc[dst], inc[x], inc[y]}
+			}
+		}
+	}
+	return is
+}
+
+var gs []float64
```

Comment:
I'm not sure.  Seemed like an artifact to me.  

https://github.com/gonum/internal/blob/master/asm/f64/scal_test.go#L213